style.css

```diff
@@ -0,0 +1,945 @@
+* {
+  margin: 0;
+  padding: 0;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #d3d3d3;
+  font-family: sans-serif;
+}
+
+body .top,
+body .Desktop-nav {
+  display: none;
+}
+
+body .mens {
+  display: none;
+  position: relative;
+  background-color: #d3d3d3;
+  width: 100%;
```

Comment:
## [OPTIONAL]

- [ ] Well done on making changes suggested by the previous reviewer but, I would really suggest you work and improve the style of this project by using the same margin, padding, colors, fontsizes, layouts, ... as in the original template.